src/syscheckd/src/db/src/fimDB.cpp

```diff
@@ -50,9 +50,9 @@ void FIMDB::init(unsigned int syncInterval,
                  std::function<void(modules_log_level_t, const std::string&)> callbackLogWrapper,
                  std::shared_ptr<DBSync> dbsyncHandler,
                  std::shared_ptr<RemoteSync> rsyncHandler,
-                 unsigned int fileLimit,
-                 unsigned int registryLimit,
-                 bool syncRegistryEnabled)
+                 const int fileLimit,
+                 const int registryLimit,
```

Comment:
There is a motivation to change this also to signed? I understand that underlying db values use signed, but populating that logic to upper layers is needed?

Reply:
It's a better way to make the type more explicit, for a module that isn't out yet.